controllers/control-plane/kyma_remote_sync_test.go

```diff
@@ -243,6 +235,76 @@ var _ = Describe("Kyma sync into Remote Cluster", Ordered, func() {
 	})
 })
 
+var _ = Describe("Kyma sync default module list into Remote Cluster", Ordered, func() {
+	kyma := NewTestKyma("kyma")
+	moduleInKCP := NewTestModule("in-kcp", v1beta2.DefaultChannel)
+	kyma.Spec.Modules = []v1beta2.Module{{Name: moduleInKCP.Name, Channel: moduleInKCP.Channel}}
+
+	var runtimeClient client.Client
+	var runtimeEnv *envtest.Environment
+
+	remoteKyma := &v1beta2.Kyma{}
+	remoteKyma.Name = v1beta2.DefaultRemoteKymaName
+	remoteKyma.Namespace = controllers.DefaultRemoteSyncNamespace
+
+	BeforeAll(func() {
+		runtimeClient, runtimeEnv = NewSKRCluster(controlPlaneClient.Scheme())
+	})
+	registerControlPlaneLifecycleForKyma(kyma)
+
+	It("Kyma CR default module list should be copied to remote Kyma", func() {
+		By("Remote Kyma created")
+		Eventually(kymaExists, Timeout, Interval).
+			WithArguments(runtimeClient, remoteKyma.Name, remoteKyma.Namespace).
+			Should(Succeed())
+
+		By("Remote Kyma contains default module")
+		Eventually(containsModuleInSpec, Timeout, Interval).
+			WithArguments(runtimeClient, remoteKyma.Name, remoteKyma.Namespace, moduleInKCP.Name).
+			Should(Succeed())
+
+		By("KCP Manifest is being created")
+		Eventually(ManifestExists, Timeout, Interval).
+			WithArguments(ctx, kyma, moduleInKCP, controlPlaneClient).
+			Should(Succeed())
+		By("KCP Manifest CR becomes ready")
```

Comment:
This step is not needed, if you need to update manifest CR manually, normally afterwards you check Kyma state become ready. But anyway in this test, it's not necessary to do this manual update.

Reply:
U r correct